docs/security_conop/evaluation_artifacts/STIG/RPM_Inspection.rst

```diff
@@ -0,0 +1,69 @@
+Verify and Correct File Permissions with RPM
+============================================
+
+* Rule ID: xccdf_org.ssgproject.content_rule_rpm_verify_permissions
```

Comment:
This part is SCAP specific as is the SSG feedback below and should be moved into the SCAP report portion.

The rest should be included in the two documents either as references or includes.